package.json

```diff
@@ -64,6 +66,7 @@
     "react-fns": "^1.4.0",
     "react-i18next": "^11.7.0",
     "react-redux": "^7.2.0",
+    "react-router": "^5.2.0",
```

Comment:
Are these new dependencies or just newly appearing in package.json? If they are new I need to run them through third-party package review.

Reply:
@davisagli The only new dependency here is `connected-react-router` (https://github.com/supasate/connected-react-router). This is not the first time I've wished we had access to the history router outside of components, but I finally decided to address it. The use-case here is wanting to do a redirect (without forcing a page-reload) in response to a redux action (in this case, a websocket event). This happens after a new scratch org has been created, if the installation is automatically started -- in which case we redirect users to the job/installation page.

Reply:
@jerivas Yeah, that's since been fixed (https://github.com/supasate/connected-react-router/issues/418) but they haven't done a new release yet. But I'm not entirely happy with how connected-react-router works anyway, and I have an idea to replace it -- I'll try that today.

Reply:
Addressed in https://github.com/oddbird/MetaDeploy/pull/314/commits/10e2baf5eb7479ac661e96d018cd3b5a9ce78d70